.github/workflows/security-snyk.yml

```
@@ -9,7 +9,7 @@ usage: "Triggered on push to main, pull requests, and manual workflow dispatch"
 behavior: "Runs Snyk security scans for code, OSS, and container, and reports findings in SARIF format"
 inputs: "Python source code, dependencies, container images"
 outputs: "SARIF reports and security labels"
-dependencies: "Snyk, Node.js, Poetry, GitHub Actions"
+dependencies: "Snyk, Node.js, uv, GitHub Actions"
```

actionlint (1.7.12)

[error] unexpected key "dependencies" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)